profiles/w3c.js

```diff
@@ -18,6 +18,7 @@ const modules = [
   import("../src/core/data-include.js"),
   import("../src/core/markdown.js"),
   import("../src/core/reindent.js"),
+  import("../src/core/title.js"),
```

Comment:
Add to geonovum profile also?

Reply:
Maintaining multiple profiles is certainly a pain 😛

Reply:
We should ask geonovum folks if they're still using the profile we maintain or their own fork.

Reply:
@sidvishnoi geonovum doesn't use the main `headers.js` plugin that w3c uses and that dini replicated. they use `fix-headers` and `id-headers` which do other things. since it seems like they don't use this feature's functionality, we didn't want to add core/title to the geonovum profile.

Reply:
Apparently, they use a self hosted version (https://github.com/search?q=respec-geonovum.js+org%3AGeonovum&type=Code) and are maintaining their own fork https://github.com/Geonovum/respec/

Reply:
That fork is superseded by the profile as maintaining a fork was too hard.

Reply:
Should've seen last commit date, instead of last open issue date :sweat_smile: